collections/_pages/about.html

```diff
@@ -5,48 +5,63 @@
-                                {%- if item.twitter %}<a class="no-ext-link-icon" href="{{ item.twitter }}"><span class="fab fa-fw fa-twitter"></span></a>{% endif %}
-                            </h5>
-                            <h6 class="card-subtitle mb-3 text-muted text-center">{{ item.position }}</h6>
-                            <div class="card-text mb-3 text-center">{{ item.summary | markdownify }}</div>
-                            {%- if item.email %}
-                            <div class="mt-auto text-center">
-                                <a class="align-self-end" href="mailto:{{ item.email }}">{{ item.email }}</a>
-                            </div>
-                            {% endif -%}
-                        </div>
-                    </div>
+<h2 id="members">{{ group.title }}</h2>
+<p class="lead">{{ group.lead }}</p>
+<div class="row mt-2">
+    {%- assign members = group.members -%}
+    {%- for item in members %}
+    <div class="{{ group.class }}">
+        <div class="card bg-transparent border-0">
+            {%- if item.img %}
+                <div class="text-center">
+                    <img class="card-img-top p-2 team-photo" src="/assets-local/team/{{item.img}}" alt="{{item.name}}">
+                </div>
+            {%- endif %}
+            <div class="card-body d-flex flex-column">
+                <h5 class="card-title text-center">
+                    <span class="pr-2">{{ item.name }}</span>
+                    {%- if item.linkedin %}<a class="no-ext-link-icon" href="{{ item.linkedin }}"><span class="fab fa-fw fa-linkedin"></span></a>{% endif %}
+                    {%- if item.twitter %}<a class="no-ext-link-icon" href="{{ item.twitter }}"><span class="fab fa-fw fa-twitter"></span></a>{% endif %}
+                </h5>
+                <h6 class="card-subtitle mb-3 text-muted text-center">{{ item.position }}</h6>
+                <div class="card-text mb-3 text-center">{{ item.summary | markdownify }}</div>
+                {%- if item.email %}
+                <div class="mt-auto text-center">
+                    <a class="align-self-end" href="mailto:{{ item.email }}">{{ item.email }}</a>
                 </div>
-                {%- endfor %}
+                {% endif -%}
             </div>
         </div>
     </div>
+    {%- endfor %}
+</div>
 {%- endfor %}
+
+</div></div>
+<div class="section"><div class="container">
+
+<h2>Support and partnerships</h2>
+<div class="lead narrow-text">
+    <p>Support and cooperation with other partners is crucial to the success of our work. In order to devote the necessary time to creating quality research, infographics and explainers, we need sufficient funding. That enables us to fully commit to making our outputs to lead to constructive discussion on climate change and related societal transformation and to data-driven decisions.</p>
```

Comment:
The last sentence sounds cumbersome to me. I'd deviate from the Czech text and write:

That also helps us communicate our findings and thus contribute to a more constructive debate on climate change and to data-driven decisions towards a net-zero society.